ts/src/profilers/profile-serializer.ts

```diff
@@ -248,6 +248,17 @@ export function serializeTimeProfile(
 export function serializeHeapProfile(
     prof: AllocationProfileNode, startTimeNanos: number,
     intervalBytes: number): perftools.profiles.IProfile {
+  // add node for external memory usage.
+  // tslint:disable-next-line: no-any
```

Comment:
1. When disabling a lint rule, add rationale in a comment. E.g. 'Current type definitions do not have external'.
2. Can you submit a PR upstream to fix this? Include a link to the PR here as a TODO here, so that the lint-disable can be removed in the future.

Reply:
Made https://github.com/DefinitelyTyped/DefinitelyTyped/pull/25580 to add `external` to `MemoryUsage`.